tasks/relayer/activateIssuer.ts

```diff
@@ -0,0 +1,24 @@
+import { task } from "hardhat/config";
+import { TaskArguments } from "hardhat/types";
+import { DefenderRelaySigner, DefenderRelayProvider } from "@openzeppelin/defender-relay-client/lib/ethers";
+import { AccessTokenVerifier } from "../../src/types";
+
+task("relay:activateIssuers", "Uses OZ Relayer to activate issuers")
+  .addParam("verifiercontract", "Address of the verifier contract")
+  .addParam("newissuer", "Address of the issuer to add")
+  .setAction(async (taskArguments: TaskArguments, hre) => {
+    if (!process.env.OZ_RELAY_API_KEY || !process.env.OZ_RELAY_API_SECRET)
+      throw new Error("Please set OZ_RELAY_API_KEY and OZ_RELAY_API_KEY env vars");
+
+    const credentials = { apiKey: process.env.OZ_RELAY_API_KEY, apiSecret: process.env.OZ_RELAY_API_SECRET };
+    const provider = new DefenderRelayProvider(credentials);
+    const signer = new DefenderRelaySigner(credentials, provider, { speed: "fast" });
```

Comment:
Do we want to parametrize the speed as well? We could be doing that on a crazy mainnet period and end up paying 100USD for gas fees

Reply:
I would say that the rotation use case here is something we would want to perform at speed, so this is fine :)